src/platform/android/NativeCodec.cpp

```diff
@@ -52,6 +59,17 @@ static jfieldID BitmapConfig_RGB_565;
 static jfieldID BitmapConfig_HARDWARE;
 
 void NativeCodec::JNIInit(JNIEnv* env) {
+  BuildVersionClass = env->FindClass("android/os/Build$VERSION");
```

Comment:
These version-checking reflection codes can be simplified by directly obtaining the high-version interface. If the attempt fails, just remember to clear the exception. You can refer to the implementation in this class: https://github.com/Tencent/tgfx/blob/main/src/platform/android/SurfaceTexture.cpp

Reply:
Done~